rules/S6756/javascript/metadata.json

```
@@ -0,0 +1,25 @@
+{
+  "title": "React state updates should not depend on stale states",
```

I was also thinking of this alternative:

```suggestion
  "title": "Calls to \"setState\" should not depend on stale states",
```

---

And also that one:

```suggestion
  "title": "\"setState\" should use a callback when referencing the previous state",
```